doc/users/prev_whats_new/whats_new_0.98.4.rst

```diff
@@ -49,18 +49,66 @@ boxes and connectors in annotations.  See
 :class:`~matplotlib.patches.ArrowStyle`, and
 :class:`~matplotlib.patches.ConnectionStyle`.
 
-.. figure:: ../../gallery/pyplots/images/sphx_glr_whats_new_98_4_fancy_001.png
-   :target: ../../gallery/pyplots/whats_new_98_4_fancy.html
-   :align: center
-   :scale: 50
+.. plot::
 
-   Whats New 98 4 Fancy
+    import matplotlib.patches as mpatch
+    import matplotlib.pyplot as plt
 
-.. _psd-amplitude:
+    figheight = 8
+    fig = plt.figure(figsize=(9, figheight), dpi=80)
```

Comment:
You can perhaps change the size here?  Dunno, but perhaps worth trying?

Reply:
I've reduced the size, but that needed more parameter changes. Now, it's a smaller version as desired. Drawbacks:

- The result is left aligned whereas all other plots are centered.
- This preserves the bad layout of the original example.

I don't care too much if we take this or my first proposal with the links to up-to-date examples.

Reply:
I wouldn't worry about the layout, the changelog entry is what it is.
The left-alignment wouldn't be a problem if we also move the other two examples  to inline in the whatsnew :p